DAEValidator/CMakeLists.txt

```diff
@@ -85,9 +85,9 @@ if (NOT LIBXML2_FOUND)
 endif ()
 
 if (WIN32)
-	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /MP")
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /MP /Wall /WX /wd4505 /wd4514 /wd4668 /wd4710 /wd4711 /wd4820")
```

Comment:
Better use pragma to only disable those warnings on third parties we cant fix

#pragma warning( push )
#pragma warning( disable : 4996 )
#include <some badly written headers> ;)
#pragma warning( pop )